docs/client/data.js

```diff
@@ -2976,7 +3157,9 @@ DocsData = {
     "summary": "Invoke a method passing an array of arguments."
   },
   "Meteor.call": {
+    "filepath": "ddp/livedata_connection.js",
```

Comment:
@mquandalle: Fair point! Although, the `Meteor.call` in `ddp/livedata_server.js` doesn't have `@summary` in its doc comment section (and doesn't have any comments at all for that matter) so the jsdoc generator just ignores it. I guess the conflict here is taken care of so only one definition of the function is passed into jsdoc. And I haven't been able to find any exception.